js/interactions.md

```diff
@@ -142,8 +142,28 @@ When using React, you can use *ChatBot* with following properties;
     welcomeMessage="Welcome, how can I help you today?"
     onComplete={this.handleComplete.bind(this)}
     clearOnComplete={true}
+    conversations={false}
 />
 ```
+The `conversations` props turns voice conversation mode on/off.
+
+If needed, you can also pass `config` in the props to modify the silence detection parameters, like in this example:
+
+```html
+const customVoiceConfig = {
+    silenceDetectionConfig: {
+        time: 2000,
+        amplitude: 0.2
+    }   
+}
+
+<ChatBot
+    ...
+    config={customVoiceConfig}
```

Comment:
This wont be available on react-native?

Reply:
correct, it won't be available on react-native